### ISSUE TITLE
[dev] improve block_filter.jq file so that `witness` field is correctly filled in case of segwit tx

### ISSUE THREAD

**Issue by TAdev0** [keep-starknet-strange/raito#95]
Currently, we have this : 

```
def txin_regular:
    "TxIn {
        script: @from_hex(\"\(.scriptSig.hex)\"),
        sequence: \(.sequence),
        previous_output: OutPoint {
            txid: 0x\(.txid)_u256.into(),
            vout: \(.vout),
            data: Default::default(),
            block_height: Default::default(),
            block_time: Default::default(),
        },
        witness: array![].span()
    }"
;
```

`witness` field should not be an empty `span<ByteArray>` in case of segwit tx, and should instead contain the witness data.

**Comment by Jeanmichel7** [comment]
Can I work on it ?

**Comment by m-kus** [comment]
Yes please

**Comment by Jeanmichel7** [comment]
@m-kus It seems that this task was already included in issue #28
